pkg/api/installopenshiftversions.go

```diff
@@ -0,0 +1,7 @@
+package api
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// InstallOpenShiftVersions represents an List of OpenShift installable version.
```

Comment:
This docstring has a typo "an List", it should read: "InstallOpenShiftVersions represents a List of OpenShift installable versions."